### ISSUE TITLE
Regression: mouse wheel unusable in scrollviewer containing stackpanel in v1.6

### ISSUE THREAD

**Issue by fredemmott** [microsoft/microsoft-ui-xaml#9979]
### Describe the bug

If a scrollviewer contains a stackpanel, the mouse wheel should be able to scroll it. This works in v1.5, but not v1.6

### Steps to reproduce the bug

1. Install MSI from https://github.com/OpenKneeboard/OpenKneeboard/actions/runs/10832469563
2. Go to settings -> advanced
3. Try to scroll with wheel
4. Does not scroll
5. Use scrollbar, scrolls

### Expected behavior

Scroll wheel should scroll.

This matches the behavior on v1.5, e.g. https://github.com/OpenKneeboard/OpenKneeboard/commit/ae0536617493f65385c2dd7df01a223c640f5edc fixes it (CI build at https://github.com/OpenKneeboard/OpenKneeboard/actions/runs/10832950077 shortly)

### NuGet package version

None

### Windows version

Windows 11 (22H2): Build 22621

### Additional context

XAML is here: https://github.com/OpenKneeboard/OpenKneeboard/blob/master/src/app/app-winui3/AdvancedSettingsPage.xaml

**Comment by Marv51** [comment]
We are also impacted by this regession. Are you able to reproduce this? What kind of additional info would be helpful to you?

**Comment by RBrid** [comment]
I have not been able to repro this issue using this markup, inspired by your  https://github.com/OpenKneeboard/OpenKneeboard/blob/master/src/app/app-winui3/AdvancedSettingsPage.xaml:

```
<Page
    ...
    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
    <ScrollViewer VerticalScrollMode="Auto"
                  Padding="24">
        <StackPanel
          Orientation="Vertical"
          Spacing="16">
            <TextBlock
              Text="Multiple Views"
              Style="{ThemeResource SubtitleTextBlockStyle}"/>
            <StackPanel Orientation="Vertical"
                        Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
                        BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
                        CornerRadius="4"
                        BorderThickness="1"
                        Padding="8"
                        Spacing="12">
                <ToggleSwitch
                  Header="Change the active view by looking"
                  OffContent="Bindings and pen strokes always affect the first view"
                  OnContent="Bindings and pen strokes affect the view you're looking at"
                  />
                <TextBlock
                  Style="{ThemeResource CaptionTextBlockStyle}"
                  Foreground="{ThemeResource TextFillColorSecondary}">
                    Both kneeboards use the same set of tabs, but track the active tab
                    and pages separately. Buttons and pen strokes always go to the
                    primary kneeboard - bind a button to switch kneeboards to control
                    the second kneeboard.
                </TextBlock>
                <ComboBox
                  Header="App window view"
                  >
                    <x:String>Ask later</x:String>
                    <x:String>In-game active view (flight sims)</x:String>
                    <x:String>Independent view (overlays, racing)</x:String>
                </ComboBox>
                <TextBlock
                  Style="{ThemeResource CaptionTextBlockStyle}"
                  Foreground="{ThemeResource TextFillColorSecondary}">
                    The app window can show and control the active in-game view
                    (for VR, this is whichever you looked at most recently), or it can
                    be independent. This changes the behavior of tab/page changes,
                    including button bindings if the OpenKneeboard window is the
                    foreground window.
                </TextBlock>
                <TextBlock
                  Style="{ThemeResource CaptionTextBlockStyle}"
                  Foreground="{ThemeResource TextFillColorSecondary}">
                    If you use tabs with multiple pages, or more tabs than views
                    (common with flight simulators), you probably want
                    'In-game active view'. Moving between tabs or pages in the app
                    window will also change the current tab or page in-game.
                </TextBlock>
                <TextBlock
                  Style="{ThemeResource CaptionTextBlockStyle}"
                  Foreground="{ThemeResource TextFillColorSecondary}">
                    If you want each view to always show the same tab, and each
                    tab only has one page (e.g. sim racing overlays), you probably
                    want 'Independent view'. Moving between tabs or pages in the app
                    window will not affect what you see in the game.
                </TextBlock>
            </StackPanel>
            <TextBlock
              Text="Mouse button bindings"
              Style="{ThemeResource SubtitleTextBlockStyle}"/>
            <StackPanel Orientation="Vertical"
                        Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
                        BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
                        CornerRadius="4"
                        BorderThickness="1"
                        Padding="8"
                        Spacing="12">
                <ToggleSwitch
                  OffContent="Disabled"
                  OnContent="Enabled"
                  />
                <TextBlock
                  Style="{ThemeResource CaptionTextBlockStyle}"
                  Foreground="{ThemeResource TextFillColorSecondary}">
                    Allows binding mouse buttons in the "Input Settings" page; this is
                    disabled by default as it is rarely used, but a common source of
                    problems.
                </TextBlock>
            </StackPanel>
            <TextBlock
              Text="Profiles"
              Style="{ThemeResource SubtitleTextBlockStyle}"/>
            <StackPanel Orientation="Vertical"
                        Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
                        BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
                        CornerRadius="4"
                        BorderThickness="1"
                        Padding="8"
                        Spacing="12">
                <ToggleSwitch
                  OffContent="Single profile"
                  OnContent="Multiple profiles"
                  />
            </StackPanel>
            <TextBlock
              Text="Bookmarks"
              Style="{ThemeResource SubtitleTextBlockStyle}"/>
            <StackPanel Orientation="Vertical"
                        Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
                        BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
                        CornerRadius="4"
                        BorderThickness="1"
                        Padding="8"
                        Spacing="12">
                <ToggleSwitch
                  OffContent="Disabled"
                  OnContent="Enabled"
                  />
            </StackPanel>
            <TextBlock
              Text="Looping"
              Style="{ThemeResource SubtitleTextBlockStyle}"/>
            <StackPanel Orientation="Vertical"
                        Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
                        BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
                        CornerRadius="4"
                        BorderThickness="1"
                        Padding="8"
                        Spacing="12">
                <ToggleSwitch
                  Header="Pages"
                  OffContent="Pages do not loop"
                  OnContent="First and last pages loop"
                  />
                <ToggleSwitch
                  Header="Tabs"
                  OffContent="Tabs do not loop"
                  OnContent="First and last tabs loop"
                  />
                <ToggleSwitch
                  Header="Profiles"
                  OffContent="Profiles do not loop"
                  OnContent="First and last profiles loop"
                  
                  />
                <ToggleSwitch
                  Header="Bookmarks"
                  OffContent="Bookmarks do not loop"
                  OnContent="First and last bookmarks loop"
                  
                  />
                <TextBlock
                  Style="{ThemeResource CaptionTextBlockStyle}"
                  Foreground="{ThemeResource TextFillColorSecondary}">
                    By default, 'Previous Page' is disabled on the first page, and
                    'Next Page' is disabled on the last page. Alternatively, pages can
                    loop so that if you're on the first page, 'Previous Page' will take
                    you to the last page, and if you're on the last page, 'Next Page'
                    will take you to the first page. Looping is similarly supported for
                    tabs, profiles, and bookmarks;
                </TextBlock>
            </StackPanel>
            <TextBlock
              Text="Tint and brightness"
              Style="{ThemeResource SubtitleTextBlockStyle}"/>
            <StackPanel Orientation="Vertical"
                        Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
                        BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
                        CornerRadius="4"
                        BorderThickness="1"
                        Padding="8"
                        Spacing="12">
                <ToggleSwitch
                  Header="Enabled"
                  OffContent="Full brightness, no tint"
                  OnContent="Custom tint and brightness"
                  />
                <Slider
                  Header="Brightness"
                  
                  TickPlacement="Outside"
                  TickFrequency="5"
                  />
                <TextBlock
                  Text="Tint color - use white for no tint"
                  Style="{ThemeResource BodyTextBlockStyle}"/>
                <ColorPicker
                  
                  ColorSpectrumShape="Ring"/>
            </StackPanel>
            <Grid ColumnDefinitions="*, Auto">
                <TextBlock
                  Grid.Column="0"
                  Text="Drawing"
                  Style="{StaticResource SubtitleTextBlockStyle}"/>
                <Button
                  Grid.Column="1"
                  Content="Restore defaults"
                  />
            </Grid>
            <StackPanel
              Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
              BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
              CornerRadius="4"
              BorderThickness="1"
              Padding="8"
              Spacing="12">
                <Slider
                  Header="Minimum pen stroke size"
                  
                  TickPlacement="Outside"
                  TickFrequency="5"
                  StepFrequency="1"
                  Maximum="50"
                  Minimum="0"/>
                <Slider
                  Header="Pen sensitivity"
                  
                  TickPlacement="Outside"
                  TickFrequency="5"
                  StepFrequency="1"
                  Maximum="50"
                  Minimum="0"/>
                <Slider
                  Header="Minimum eraser size"
                  
                  TickPlacement="Outside"
                  TickFrequency="50"
                  StepFrequency="1"
                  Maximum="300"
                  Minimum="0.0"/>
                <Slider
                  Header="Eraser sensitivity"
                  
                  TickPlacement="Outside"
                  TickFrequency="100"
                  StepFrequency="1"
                  Maximum="500"
                  Minimum="1.0"/>
            </StackPanel>
            <Grid ColumnDefinitions="*, Auto">
                <TextBlock
                  Grid.Column="0"
                  Text="Text"
                  Style="{StaticResource SubtitleTextBlockStyle}"/>
                <Button
                  Grid.Column="1"
                  Content="Restore defaults"
                  />
            </Grid>
            <StackPanel
              Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
              BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
              CornerRadius="4"
              BorderThickness="1"
              Padding="8"
              Spacing="12">
                <Slider
                  Header="Font size"
                  
                  TickPlacement="Outside"
                  TickFrequency="5"
                  StepFrequency="1"
                  Maximum="30"
                  Minimum="6"/>
            </StackPanel>
            <TextBlock
              Text="In-Game Interface"
              Style="{ThemeResource SubtitleTextBlockStyle}"/>
            <StackPanel Orientation="Vertical"
                        Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
                        BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
                        CornerRadius="4"
                        BorderThickness="1"
                        Padding="8"
                        Spacing="12">
                <ToggleSwitch
                  Header="Header and toolbar"
                  OffContent="No in-game header or toolbar"
                  OnContent="In-game header and toolbar are shown"
                  />
                <ToggleSwitch
                  Header="Footer and clock"
                  OffContent="No in-game footer or clock"
                  OnContent="In-game footer and clock are shown"
                  />
                <ToggleSwitch
                  Header="Frame counter in footer"
                  OffContent="Frame counter is not shown"
                  OnContent="The number of OpenKneeboard frames is shown"
                  
                  />
            </StackPanel>
            <Grid ColumnDefinitions="*, Auto">
                <TextBlock
                  Grid.Column="0"
                  Text="Compatibility quirks"
                  Style="{StaticResource SubtitleTextBlockStyle}"/>
                <Button
                  Grid.Column="1"
                  Content="Restore defaults"
                  />
            </Grid>
            <StackPanel
              Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
              BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
              CornerRadius="4"
              BorderThickness="1"
              Padding="8"
              Spacing="12">
                <ComboBox
                  Header="Elevation"
                  
                  >
                    <x:String>Run elevated if launched elevated</x:String>
                    <x:String>Run elevated</x:String>
                    <x:String>Run as normal user</x:String>
                </ComboBox>
                <TextBlock
                  Style="{ThemeResource CaptionTextBlockStyle}"
                  Foreground="{ThemeResource TextFillColorSecondary}">
                    This reduces performance, but can workaround flickering issues
                    caused by bugs in other OpenXR components.
                </TextBlock>
                <ToggleSwitch
                  Header="Legacy Oculus API (LibOVR): Discard depth information"
                  OffContent="Disabled - keep depth information"
                  OnContent="Enabled - discard depth information"
                  />
                <TextBlock
                  Style="{ThemeResource CaptionTextBlockStyle}"
                  Foreground="{ThemeResource TextFillColorSecondary}">
                    If the OpenKneeboard is not visible in games using the legacy Oculus
                    API (LibOVR), this may fix it; however, when things work correctly
                    without it, it may significantly reduce the quality of reprojection.
                </TextBlock>
                <TextBlock
                  Style="{ThemeResource CaptionTextBlockStyle}"
                  Foreground="{ThemeResource TextFillColorSecondary}">
                    When this option helps, it indicates a bug in the game, the active
                    OVR runtime, or another mod.
                </TextBlock>
                <ComboBox
                  Header="OpenXR: Upscale images"
                  >
                    <x:String>Automatic (Varjo only)</x:String>
                    <x:String>Leave for OpenXR runtime</x:String>
                    <x:String>Always</x:String>
                </ComboBox>
                <TextBlock
                  Style="{ThemeResource CaptionTextBlockStyle}"
                  Foreground="{ThemeResource TextFillColorSecondary}">
                    Increases the resolution of content before passing to the OpenXR
                    runtime; runtimes are theoretically able to produce better results
                    than OpenKneeboard can, but in practice OpenKneeboard produces better
                    results than some vendors.
                </TextBlock>
            </StackPanel>
        </StackPanel>
    </ScrollViewer>
</Page>

```

I also installed the app and went through the steps:
- Install MSI from https://github.com/OpenKneeboard/OpenKneeboard/actions/runs/10832469563
- Go to settings -> advanced
- Try to scroll with wheel
No repro either.  So I'm closing this issue.

**Comment by fredemmott** [comment]
Do you have suggestions for things we could look into to narrow this down? It is somewhat disheartening to have a regression ignored for months then closed with "works on my machine".

**Comment by fredemmott** [comment]
Regression still exists, still fixed by reverting to v1.5. Now affects WinUI3 Control Gallery and Microsoft Terminal from the store.

Tried:
- switching to an extremely basic mouse
- reducing DPI scaling to 100%
- reducing monitor refresh rate to 30hz

**Comment by fredemmott** [comment]
This is the same as #10091 - it is fixed by changing this setting: 

<img width="1015" alt="Image" src="https://github.com/user-attachments/assets/62098b3f-4a9a-4a3a-b429-8153ab98afd4" />
